options/locale/locale_en-US.ini

```diff
@@ -1358,6 +1358,16 @@ issues.filter_milestones = Filter Milestone
 issues.filter_projects = Filter Project
 issues.filter_labels = Filter Label
 issues.filter_reviewers = Filter Reviewer
+issues.filter = Filter
+issues.filter.reset = Reset Filter
+issues.filter.is_archived = Archived
+issues.filter.not_archived = Not Archived
+issues.filter.is_fork = Forked
+issues.filter.not_fork = Not Forked
+issues.filter.is_mirror = Mirrored
+issues.filter.not_mirror = Not Mirrored
+issues.filter.public = Public
+issues.filter.private = Private
```

Comment:
It's a recurring problem with the structure of the translations, but I would recommend moving these to top-level in the ini so they are re-usable in the future. Nothing about these strings is specific to issue filters:

```
filter = Filter
reset_filter = Reset Filter
is_archived = Archived
not_archived = Not Archived
is_fork = Forked
not_fork = Not Forked
is_mirror = Mirrored
not_mirror = Not Mirrored
public = Public
private = Private
```

Reply:
Done with https://github.com/go-gitea/gitea/pull/29231/commits/3c386fd3a0b329e458e80b418b0fc72936aec47e

Reply:
Nice, thanks.